### PR TITLE
Updated bracken to v2.8

### PR DESCRIPTION
**Pull request by tuncK**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/bracken/est-abundance.xml [hunk 1]
```diff
@@ -1,20 +1,25 @@
-<tool id="est_abundance" name="Estimate Abundance at Taxonomic Level" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
-    <description>Bayesian Reestimation of Abundance with Kraken</description>
+<tool id="est_abundance" name="bracken" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
```
[the remark this answers is not in the document]
Comment (tuncK) [reply tools/bracken/est-abundance.xml:1]: Yes, hence I changed the "name". But if I change tool_id itself I guess the connection between the versions will be lost.

[the remark this answers is not in the document]
Comment (bgruening) [reply tools/bracken/est-abundance.xml:1]: But shouldn't the name be `Bracken` with capital B?

[the remark this answers is not in the document]
Comment (tuncK) [reply tools/bracken/est-abundance.xml:1]: Haaa, now got it.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @tuncK

**Pull request merged**